iOS/SonarKit/SKMacros.h

```diff
@@ -9,7 +9,7 @@
  #ifndef SKMACROS_H
  #define SKMACROS_H
 
- #import <FBDefines/FBMacros.h>
```

Comment:
Will break internal compatibility for us. Please read the above comment

Reply:
Hm, yeah I ended up doing this since generally we're importing files from the same target using quotes rather than angle brackets, but I was kind of worried about this breaking systems that don't use the external podspec. I'll try and find a way to make the header paths work correctly here.